### ISSUE TITLE
Project Staring / Favoriting

### ISSUE THREAD

**Issue by cycomachead** [snap-cloud/snap-cloud#109]
Projects should have a way of being starred / favorited

- Users should see the projects they have favorited
- project pages should show a count of favs / stars
- there should be a page to see all the people who have starred the project

Also, we need to decide on a vocabulary for this feature

**Comment by cycomachead** [comment]
@brianharvey I might work on this one soon for the Snap<i>!</i> Cloud. What would your preferred vocabulary be for this? Scratch currently has both favorites and hearts ("love") which I personally find a little confusing, and I think we should start out as having only one option.

Other sites for reference:
- Github: Stars
- Twitter: Stars
- Tumblr: Hearts
- Facebook: thumbs up / like
- Instagram: hearts
- Pinterest: hearts
- Google+: +1s

**Comment by brianharvey** [comment]
Yeah, at /most/ one.  I'm leaning toward zero, but I guess I'm not a kid.  If we have to have one, let's go for "favorite," because I think that doesn't sound so much as if you're supposed to click it for every project you look at.
